packages/fx-core/src/core/FxCore.ts

```diff
@@ -268,6 +272,51 @@ export class FxCore implements v3.ICore {
     return this.provisionResources(inputs);
   }
 
+  @hooks([
+    ErrorHandlerMW,
+    ConcurrentLockerMW,
+    ProjectMigratorMW,
+    ProjectConsolidateMW,
+    AadManifestMigrationMW,
+    ProjectVersionCheckerMW,
+    ProjectSettingsLoaderMW,
+    EnvInfoLoaderMW_V3(false),
+    ContextInjectorMW,
+    EnvInfoWriterMW_V3(),
+  ])
+  async deployAadManifest(inputs: Inputs, ctx?: CoreHookContext): Promise<Result<Void, FxError>> {
+    setCurrentStage(Stage.deployAad);
+    inputs.stage = Stage.deployAad;
+    const updateAadClient = Container.get("aadApp/update") as any;
+    // In V3, the aad.template.json exist at .fx folder, and output to root build folder.
+    const manifestTemplatePath: string = path.join(inputs.projectPath!, ".fx", "aad.template.json");
```

Comment:
a heads up that `aad.template.json` will be moved out of `.fx` folder

Reply:
Will update the path after settled